drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml

```diff
@@ -71,6 +71,8 @@ spec:
           value: "false"
         - name: "ES_JAVA_OPTS"
           value: "-Xms256m -Xmx256m"
+        - name: "NETWORK_HOST"
+          value: "_eth0_"
```

Comment:
do you have a reference online that suggests to use this?

Reply:
I found it here and worked for our case https://github.com/pires/kubernetes-elasticsearch-cluster/issues/64